src/pages/Subscribe/Subscribe.js

```diff
@@ -1,7 +1,362 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { useNavigate, useLocation } from 'react-router-dom';
 
 const Subscribe = () => {
-  return <h1>Subscribe</h1>;
+  const { pathname } = useLocation();
+
+  const [index, setIndex] = useState(1);
+  const [name, setName] = useState('');
+  const [address, setAddress] = useState('');
+  const PRICE = 39000;
+
+  const result = PRICE * index;
```

Comment:
```suggestion
  const totalPrice = PRICE * userInfo.index;
```
음..result보단 totalPrice 혹은 최종가격에 finalPrice 정도는 어떨까요?